clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt

```diff
@@ -61,59 +59,61 @@ interface VulnerableCodeService {
     }
 
     /**
-     * Data class to represent a reference to a vulnerability. When querying the vulnerabilities for packages, each
-     * package in the result has a list of this type. In order to retrieve further details about vulnerabilities,
-     * an additional request has to performed.
+     * Data class that represents a score assigned to a vulnerability. A source of vulnerability information can
+     * provide multiple score values using different scoring systems.
      */
-    data class HyperLinkedVulnerability(
-        /** A URL with information about the vulnerability. */
-        val url: String,
+    data class Score(
+        /** The name of the scoring system. */
+        val scoringSystem: String,
 
-        /**
-         * The ID of this vulnerability, typically a CVE identifier. According to the VulnerableCode schema, this can
-         * be *null*.
-         */
-        val vulnerabilityId: String?
+        /** The value in this scoring system. */
```

Comment:
Nit: Maybe briefly explain here why this is a string? (Because a value might also be something like "Low".)